pkgdown/_pkgdown.yml

```diff
@@ -51,23 +51,25 @@ articles:
   - starts_with("test")
 
 reference:
-- title: Build
-  desc:  Build a complete site or one of its components.
+- title: Site components
+  desc:  Build a complete site or one of its components. See `vignette("pkgdown")`
+    for an overview of site components.
   contents:
   - build_site
   - starts_with("build_")
   - -build_site_github_pages
-  - init_site
-  - preview_site
   - pkgdown_sitrep
 
-- title: Deployment
+- title: Deployment and Build
+  desc: Build or deploy the site.
   contents:
+  - init_site
```

Comment:
I do think `init_site()` belongs above since the css that it builds is a site "component"